src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,257 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.model.User;
+import core.basesyntax.service.model.User;
+import org.junit.jupiter.api.BeforeAll;
```

Comment:
Consider using `@BeforeEach` instead of `@BeforeAll` for initializing `registrationService`. `@BeforeEach` runs before each test, ensuring that tests do not interfere with each other, while `@BeforeAll` runs once before all tests, which could potentially lead to shared state between tests.